README.md

```diff
@@ -41,6 +41,8 @@ creative guide.
 - [ ] Share your board with the project manager that has been assigned to you. If you have not been assigned yet, reach out to your lead PM for guidance
 - [ ] Add your Trello URL to your project's README.md file. Commit the change, push it to your repository & submit a pull request
 
+https://trello.com/b/6Sp67zaD/lambdamud-tsai
```

Comment:
Good that you put your trello link in here. It's a small thing but a lot of students forget to do this, but it's really important for communicating your work so great job.